examples/models/models_first_person_maze.cpp

```diff
@@ -25,7 +25,7 @@ int main(void)
 
     raylib::Image imMap("resources/cubicmap.png");      // Load cubicmap image (RAM)
     raylib::Texture cubicmap(imMap);                    // Convert image to texture to display (VRAM)
-    Mesh mesh = raylib::Mesh::Cubicmap(imMap, Vector3{ 1.0f, 1.0f, 1.0f });
+    raylib::MeshUnmanaged mesh = raylib::MeshUnmanaged::Cubicmap(imMap, Vector3{ 1.0f, 1.0f, 1.0f });     // Use MeshUnmanaged, Mesh will be handled by Model
     raylib::Model model(mesh);
```

Comment:
Thanks for taking this on! Definitely does make sense since Model ends up taking ownership of the Model.

In addition to having `MeshUnmanaged`, I think having an overloaded method for `raylib::Model(raylib::Mesh)` that creates a copy of the Mesh as MeshUnmanaged, and uses that. This way the example code wouldn't need to change, but memory would still be :ok_hand:

Reply:
I'll merge this forwards, and then we can take it on as a follow up PR.